### PR TITLE
Bug 1149016 - allow spaces in paths for --ssh flag but still allow argum...

### PR DESCRIPTION
**Pull request by jwforres**
...ents.

Executable path must be quoted within the string with escaped quotes if it contains spaces in the path

### CODE REVIEW

**Comment by jwforres** [comment]
@liggitt @fabianofranz

**Comment by liggitt** [comment]
LGTM

File: lib/rhc/commands/ssh.rb [hunk 1]
```diff
@@ -15,6 +15,10 @@ class Ssh < Base
 
       You may run a specific SSH command by passing one or more arguments, or use a
       different SSH executable or pass options to SSH with the '--ssh' option.
+
+      To use the '--ssh' flag with an SSH executable path containing a space, wrap
+      the executable path with double quotes:
+      --ssh '"path with spaces\\ssh"'
```
Comment (fabianofranz) [inline lib/rhc/commands/ssh.rb]: Either
--ssh '"/path with spaces/ssh"' or
--ssh '"C:\\path with spaces\ssh"' if you mean Windows

**Comment by fabianofranz** [comment]
LGTM

**Comment by jwforres** [comment]
[merge]

**Comment by jwforres** [comment]
unrelated [merge]

**Pull request merged**